cypress/TestCases/FireboltCertification/SecureStorage.feature

```diff
@@ -224,4 +224,115 @@ Feature: SecureStorage
\ No newline at end of file
+            | without scope                 | clear stored value without scope              |
+
+    # clears the existing key for 1st app
+    # 2nd app is launched
+    # clears the existing key for 2nd app
+    # validates the cleared value for 1st app
+    # validates the cleared value for 2nd app
+    # set value1 in device scope
+    # validates the set values for 1st app
+    # validates the set value for 2nd app
+    @Securestorage @coreSDK @sdk @transport
+    Scenario Outline: SecureStorage.set - Positive Scenario: Validate <Scenario>
+        Given '3rd party app' invokes the 'Firebolt' API to '<Clear_API_Key>'
+        And 'Firebolt' platform responds with 'null for clearing stored value'
+        And 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        And 'secondary 3rd party app' invokes the 'Firebolt' API to '<Clear_API_Key>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with 'null for clearing stored value'
+        And '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key>'
+        And 'Firebolt' platform responds with 'null for getting stored value'
+        And 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with 'null for getting stored value'
+        When '3rd party app' invokes the 'Firebolt' API to '<Set_API_Key>'
+        Then 'Firebolt' platform responds with 'null for updating a secure data value'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key>'
+        Then 'Firebolt' platform responds with '<Validation_Key>'
+        And 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with '<Validation_Key>'
+
+        Examples:
+            | Scenario                                    | Get_API_Key                                                             | Set_API_Key                                                    | Validation_Key                                                       | Clear_API_Key                            |
+            | Adding scope as device in 2 apps             | get stored value with scope as device and key as authTestTokenDevice    | update stored value for key authTestTokenDevice                | expected value for authTestTokenDevice stored data in securestorage  | clear stored value with scope as device  |
+
+    # 2nd app is launched
+    # set value1 in device scope
+    # validates the set values for 1st app
+    # set value2 in device scope
+    # validates the set values for 1st app
+    # validates the set values for 2nd app
+    # removes value1 with device scope
+    # validates the removed value for 1st app
+    # validates the removed value for 2nd app
+    # validates the remaining value for 1st app
+    # validates the remaining value for 2nd app
```

Comment:
not sure if we are allowed to give And before Given in BDD. Check and update properly